test/assert.js

```diff
@@ -237,3 +283,122 @@ QUnit.test( "raises, alias for throws", function( assert ) {
 		throw "my error";
 	});
 });
+
+QUnit.module( "failing assertions", {
+	beforeEach: function( assert ) {
+		assert.push = function( result, actual, expected, message ) {
+
+			// inverts the result so we can test failing assertions
+			QUnit.assert.push.call( assert, !result, actual, expected, message );
```

Comment:
As the QUnit.assert is a reference for Assert's prototype and we can just call it with assert, I believe it's better than storing a variable in the memory, also it takes less loc.